tests/ops/integration_tests/setup_scripts/postgres_setup.py

```diff
@@ -12,6 +12,10 @@
     ConnectionConfig,
     ConnectionType,
 )
+
+# Need to manually import this model because it's used in src/fides/api/models/property.py
+# but that file only imports it conditionally if TYPE_CHECKING is true
+from fides.api.models.privacy_experience import PrivacyExperienceConfig
```

Comment:
this change was needed so the `postgres_setup` script would run correctly, but is technically unrelated to the pymongo upgrade